addons/overheating/functions/fnc_jamWeapon.sqf

```diff
@@ -5,19 +5,20 @@
  *
  * Arguments:
  * 0: Unit <OBJECT>
- * 1: Weapon <STRING>
+ * 1: Weapon <STRING>, (default: primaryWeapon ace_player)
+ * 2: Jam Type <STRING> (default: ""), must be an allowed jam type for the weapon, if default or invalid selects randomly from allowed types
  *
  * Return Value:
  * None
  *
  * Example:
- * [player, currentWeapon player] call ace_overheating_fnc_jamWeapon
+ * [player, primaryWeapon player, "Feed"] call ace_overheating_fnc_jamWeapon
  *
  * Public: No
  */
 
-params ["_unit", "_weapon"];
-TRACE_2("params",_unit,_weapon);
+params ["_unit", ["_weapon", primaryWeapon ace_player, ""], ["_jamType", "", ""]];
```

Comment:
I don't like the default being `primaryWeapon ace_player`, I'd prefer if `param` were to be used further below:

`private _weapon = param [1, primaryWeapon _unit, [""]];`